### DISCUSSION TITLE
How to get full path of selected Cascade? onChange

### DISCUSSION THREAD

**Discussion by george-of-314** [rsuite/rsuite#2850] [Q&A] [answered]
![зображення](https://user-images.githubusercontent.com/9105235/199297974-605b21ec-963a-46d0-8336-669d32264c66.png)

How to get full path of selected Cascade using onChange property?
I need "Sichuan / Zigong / Gongjing", like on screenshot

Thank you

**Comment by simonguo** [comment, marked as answer]
You can use `onSelect` to get the full path.

**Comment by george-of-314** [comment]
Thanks. Used this

But too complicated. Why not use with onChange?

**Reply by simonguo** [reply]
Because the callback parameters of onChange should be consistent with other form components, so that form validation can be better processed uniformly.
https://rsuitejs.com/components/form-validation/
